satpy/tests/reader_tests/conftest.py

```diff
@@ -15,4 +15,243 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
+
 """Setup and configuration for all reader tests."""
+
+import os
+from datetime import datetime, timedelta
```

Comment:
Please use `import datetime as dt` so that the imported `datetime` module does not make `datetime` package unaccesible.